ports/lhasa/vcpkg.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "lhasa",
+  "version": "0.4.0",
+  "description": "LZH/LHA archive library",
+  "homepage": "https://fragglet.github.io/lhasa/",
+  "license": "ISC"
```

```suggestion
  "license": "ISC",
  "dependencies": [
    {
      "name": "pkgconf",
      "host": true,
      "default-features": false
    }
  ]
```
Please add the dependencies to use the pkg-config.

---

@FrankXie05 Why?

---

For the usage.

---

For using this port? (I have pkg-config.) 
Or for building this port? (Don't we we still use `vcpkg_find_acquire_program(PKGCONFG)`?)

---

Of course this method can be used, what I mean is that this port requires `pkgconfg` to be installed. :)

---

> Of course this method can be used, what I mean is that this port requires `pkgconfg` to be installed. :)

I don't see why it requires **port `pkgconf`**.

---

Or add `vcpkg_find_acquire_program(PKGCONFIG)` to `portfile.cmake`.